addons/mail/static/src/js/rotting_mixin/rotting_widget.js

```diff
@@ -46,22 +44,28 @@ export class KanbanRottingField extends Component {
     }
 }
 
-export class ListBadgeSelectionRotting extends ListBadgeSelectionField {
-    static template = "mail.ListBadgeSelectionRotting";
+export class Many2OneFieldRotting extends Component {
```

Comment:
Why not directly extend Many2OneField then? You essentially are doing an extend without actually extending the class.